### PR TITLE
snyk 1.136.1 (new formula)

### PR DESCRIPTION
**Pull request by darmalovan**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by SMillerDev** [comment]
Homebrew core doesnt ship pre-built binaries like these. This formula will need to be rewritten to use the `npm install -g snyk` way of installing (look for other npm tools for an idea how.)

**Comment by darmalovan** [comment]
Thanks @SMillerDev, I took an example from the heroku-cli.
I'm trying to find a good example for a npm package.

**Comment by darmalovan** [comment]
@SMillerDev - I think I found the right way to do it, will appreciate your review again when you have time.

File: Formula/snyk.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+require "language/node"
+
+class Snyk < Formula
+  desc "CLI and build-time tool to find & fix known vulnerabilities in open-source dependencies"
+  homepage "https://snyk.io"
+  url "https://registry.npmjs.org/snyk/-/snyk-1.136.1.tgz"
+  version "1.136.1"
```
Comment (SMillerDev) [inline Formula/snyk.rb]: This will be picked up from the URL.

File: Formula/snyk.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    system bin/"snyk", "version"
```
Comment (SMillerDev) [inline Formula/snyk.rb]: Could you try and do a more meaningful test then this? Version checks rarely give a real indication of a failing build.

**Review by SMillerDev — CHANGES_REQUESTED** [review]

File: Formula/snyk.rb (hunk 2 above)
Comment (darmalovan) [reply Formula/snyk.rb]: Thanks, pushing the change!

Comment (darmalovan) [reply Formula/snyk.rb]: Got delayed a bit, will probably get back to it tomorrow, or next week.
Really appreciate the responsiveness here :)

**Comment by darmalovan** [comment]
@SMillerDev changed the test.

**Comment by equal-l2** [comment]
@darmalovan `brew audit` fails.

```
Error: 1 problem in 1 formula detected
snyk:
   * C: 4: col 3: Description is too long. "name: desc" should be less than 80 characters. Length is calculated as snyk + desc. (currently 93)
```

Would you replace the description with a more concise one?

**Comment by darmalovan** [comment]
Addressed @equal-l2

**Comment by equal-l2** [comment]
@darmalovan Thank you for the quick fix! 
Unfortunately, I don't have merge permission, so please wait a bit for appropriate people to merge.

**Comment by darmalovan** [comment]
@SMillerDev how can we proceed?

**Comment by darmalovan** [comment]
Closing - in order to verify how updates are shown for brew installation.

**Pull request closed without merging**